lib/experimental/enqueue-webfonts-listed-in-theme-json.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Emit inline CSS for allowed webfonts listed in theme.json.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Enqueue webfonts listed in theme.json.
+ *
+ * Enqueued webfonts will end up in the front-end as inlined CSS.
+ */
+function _gutenberg_enqueue_webfonts_listed_in_theme_json() {
```

Comment:
Should we prefix this function with `_wp` instead of `_gutenberg`? This will end up in WP Core at some point.